src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <TimeZoneDataVersion>2020a</TimeZoneDataVersion>
+      <TimeZoneSource>https://data.iana.org/time-zones/tzdb</TimeZoneSource>
+      <InputFolder>$(MSBuildProjectDirectory)/input</InputFolder>
+      <OutputFolder>$(MSBuildProjectDirectory)/zoneinfo</OutputFolder>
+    </PropertyGroup>
+    <ItemGroup>
+        <Area Include="africa" />
+        <Area Include="antarctica" />
+        <Area Include="asia" />
+        <Area Include="australasia" />
+        <Area Include="etcetera" />
+        <Area Include="europe" />
+        <Area Include="northamerica" />
+        <Area Include="southamerica" />
+    </ItemGroup>
+    <ItemGroup>
+    </ItemGroup>
+    <Target Name="DownloadData" BeforeTargets="Build" Condition="!Exists('$(OutputFolder)')">
```

Comment:
@tqiu8 I think I'd prefer if we split downloading the input files and running `zic` into two steps, with the download step being manual.

so something like this:

```xml
    <Target Name="UpdateData">
        <DownloadFile
            SourceUrl="$(TimeZoneSource)-$(TimeZoneDataVersion)/%(Area.Identity)"
            DestinationFolder="$(InputFolder)">
        </DownloadFile>
        <DownloadFile
            SourceUrl="$(TimeZoneSource)-$(TimeZoneDataVersion)/zone1970.tab"
            DestinationFolder="$(OutputFolder)">
        </DownloadFile>
    </Target>

    <Target Name="CompileZoneFile" BeforeTargets="Build">
        <Exec Command="zic -d $(OutputFolder) $(InputFolder)/%(Area.Identity)" />
    </Target>
```

that will allow us to check in the input files which are text files (easier for git) and create the binary files as part of the build.